### ISSUE TITLE
Properly report errors when an invalid media/stream URI is specified

### ISSUE THREAD

**Issue by jrobble** [openmpf/openmpf#211]
Make sure that an error is properly reported when running a batch job with an invalid media URI or a streaming job with an invalid stream URI.

In the former case, the `status` and `message` fields of the JSON output are updated as follows:

```
"media": [{
    "mediaId": 13,
    "path": "bad:///samples/face/video.mp4",
    "mimeType": null,
    "length": 0,
    "sha256": null,
    "message": "Unsupported URI sceheme.",
    "status": "ERROR",
    "mediaMetadata": {},
    "mediaProperties": {},
    "markupResult": null,
    "output": {
      "NO TRACKS": [{
        "source": "+#DLIB FACE DETECTION ACTION",
        "tracks": []
      }, {
        "source": "+#DLIB FACE DETECTION ACTION#OCV GENERIC MARKUP ACTION",
        "tracks": []
      }]
    },
    "detectionProcessingErrors": {}
  }]
```

However, because `detectionProcessingErrors` is not populated, the job appears to complete without errors in the web UI.

Minimally, the job status should be updated to JobStatus.IN_PROGRESS_ERRORS in Redis.

**Comment by jrobble** [comment]
@dpierce17, FYI.

**Comment by jrobble** [comment]
Releted to #171.

**Comment by jrobble** [comment]
This is currently being addressed in PR #250.

**Comment by dpierce17** [comment]
Closed, landed in PR #250
